intuita-webview/src/codemodList/CodemodNodeRenderer/index.tsx

```diff
@@ -88,13 +83,9 @@ const getCodemodNodeRenderer =
 					/>
 				)}
 
-				{node.kind === 'DIRECTORY' && (
+				{['DIRECTORY', 'ROOT'].includes(node.kind) ? (
```

Comment:
do we need the ternary operator here? I think `&&` should work

Reply:
some styleguides recommend always use trenary, because if we use some falsy values like 
```
let count = 0;
...
{ count && <div /> }
```
it will render `0`

Reply:
but in this case we can use &&